modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding the {product-title} container images
+You can embed {product-title}'s container images into the {op-system-ostree} images, so that they are immediately available to the CRI-O container engine after booting, without having to pull the images over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+.Example adding a container image to a blueprint:
+[source,terminal]
+----
+[[containers]]
+source = "<image_pullspec_with_tag_or_digest>"
+----
+
+To embed {product-title}'s container images, you need to know the exact list of container image references used by the {product-title} version you are embedding. You can get this list from the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint using the following procedure.
+
+You need to have the exact list of container image references used by the {product-title} version to embed {product-title} 's container images. Use the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint.
+
+.Procedure
+
+. Download the `microshift-release-info` RPM package matching your {product-title} version running the following command:
++
+[source,terminal]
+----
+$ yum download microshift-release-info-4.12.0-1.el8
+----
+
+. Exctract the release info by running the following command:
++
+[source,terminal]
+----
+$ rpm2cpio microshift-release-info-4.12.0-1.el8.noarch.rpm | cpio -idmv
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
+----
+
+. Generate the lines to append to your blueprint using the release info for your CPU architecture by running the following command:
++
+[source,terminal]
+----
+$ jq -r '.images | .[] | ("[[containers]]\nsource = \"" + . + "\"\n")' ./usr/share/microshift/release/release-$(uname -i).json
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9945c3f5475a37e145160d2fe6bb21948f1024a856827bc9e7d5bc882f44a750"
+
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:82cfef91557f9a70cff5a90accba45841a37524e9b93f98a97b20f6b2b69e5db"
+
```

Comment:
Let's separate the command from the file contents?

Reply:
@ggiguash I don't understand what you'd like changed here.

Reply:
This is the command:
```
$ jq -r '.images | .[] | ("[[containers]]\nsource = \"" + . + "\"\n")' ./usr/share/microshift/release/release-$(uname -i).json
```

This is a sample output:
```
[[containers]]
source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9945c3f5475a37e145160d2fe6bb21948f1024a856827bc9e7d5bc882f44a750"
[[containers]]
source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:82cfef91557f9a70cff5a90accba45841a37524e9b93f98a97b20f6b2b69e5db"
```

Can we have those separated?

Reply:
Got it, thank you!